soh/src/overlays/actors/ovl_Boss_Dodongo/z_boss_dodongo.c

```diff
@@ -1697,7 +1697,7 @@ void BossDodongo_DrawEffects(GlobalContext* globalCtx) {
     func_80093D84(globalCtx->state.gfxCtx);
     unkMtx = &globalCtx->billboardMtxF;
 
-    gSPInvalidateTexCache(POLY_XLU_DISP++, 0);
+    // gSPInvalidateTexCache(POLY_XLU_DISP++, 0);
```

Comment:
Not sure we want to do this. It may break the scene texture change. It may also break the changing of texture of king doodong himself. Can you test that this still works?